src/components/banner/template.njk

```diff
@@ -1,5 +1,23 @@
 <div class="hmrc-banner">
   <div class="hmrc-organisation-logo">
+  <!--[if gt IE 8]><!-->
+    <svg id="f2a14da0-d308-46dc-ad18-2464344bd5d1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" width="26px" height="26px" viewBox="0 0 134.94 134.94">
```

Comment:
@tomshaw1 I wonder if we should include this via linking to the svg file here rather than having it defined inline. We've got the file in the PR separately but then duplicated here. If we have it separate then it can be cached by people's browsers, only 7kb, but that 7kb extra will be sent with every request rather than cached because the html won't be being cached by browsers in most cases

Reply:
Hey @oscarduignan, the reason we did it inline is that GOV.UK does the same, which Tim found here https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/components/header/template.njk#L8-L19
I agree that it shouldn't be duplicated, so I'll delete the .svg from the images file.